components/Article/ReportageFormat.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import ActionBar from '../ActionBar/'
+import Author from './Author'
+import Time from './Time'
+
+import { css } from 'glamor'
+
+import { Link } from '../../routes'
+import { PUBLIC_BASE_URL } from '../../constants'
+
+import {
+  H1,
+  H2,
+  Interaction,
+  Label,
+  Lead,
+  P,
+  mediaQueries
+} from '@project-r/styleguide'
+
+// TODO: Implement light/dark theme.
+// TODO: Implement photo for cover.
+
+const styles = {
+  cover: css({
+    backgroundColor: '#ddd',
+    minHeight: '500px',
+    padding: '50px 20px 180px 20px',
+    position: 'relative',
+    textAlign: 'center',
+    [mediaQueries.onlyS]: {
+      margin: '0 -20px'
+    }
+  }),
+  meta: css({
+    bottom: '20px',
+    left: 0,
+    position: 'absolute',
+    right: 0
+  })
+}
+
+const ReportageFormat = ({ article }) => {
+  return (
+    <article>
+      <div {...styles.cover}>
+        <H1 style={{ textAlign: 'center' }}>
+          {article.title}
+        </H1>
+        <div {...styles.meta}>
+          <Author name={article.author} showBadge={true} />
```

Comment:
`<Author name={article.author} showBadge />`
is the short hand notation for a `true` bool prop and equivalent to:
`<Author name={article.author} showBadge={true} />`